src/main/java/com/rtosProject2/Gui.java

```diff
@@ -25,33 +25,36 @@ public static void main(String[] args) throws IOException, InterruptedException
         Console console = new Console();
         Display display = ConfigureDisplay();
 
-        DoubleBuffer<String[][][]> bufferAB = new DoubleBuffer<>(1);
-        DoubleBuffer<Object[][]> bufferCD = new DoubleBuffer<>(1);
+        DoubleBuffer<Message> bufferAB = new DoubleBuffer<>(1);
+        DoubleBuffer<Message> bufferCD = new DoubleBuffer<>(1);
 
         //TO SINGLE STEP, SET delayMs TO 0.
-        int delayMs = 1000;
+        int delayMs = 5;
```

Comment:
Why do we have a change in time here?

Reply:
Sorry - shouldn't have checked that in. I was running it at 5ms to test it through 200 iterations.